home/models.py

```diff
@@ -295,7 +295,29 @@ def clean(self, value):
                 f"{self.MEDIA_CAPTION_MAX_LENGTH} characters, your message is "
                 f"{len(result['message'])} characters long"
             )
+
+        # find variables
+        vars_in_msg = re.findall(r"{{(.*?)}}", result["message"])
+        non_digit_variables = [var for var in vars_in_msg if not var.isdigit()]
```

Comment:
I think `isdigit` will match on things we don't want, like superscripts and Kharosthi numbers: https://docs.python.org/3/library/stdtypes.html#str.isdigit . I don't think WhatsApp templates support those.

Maybe we want `isdecimal` instead? https://docs.python.org/3/library/stdtypes.html#str.isdecimal